interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -63,26 +109,26 @@
 // Global filtering.
 //
 $where = '';
+$sSearch = add_escape_custom(trim($_GET['sSearch']));
+$cnt_columns = count($aColumns);
+
 if (isset($_GET['sSearch']) && $_GET['sSearch'] !== "") {
-    $sSearch = add_escape_custom(trim($_GET['sSearch']));
-    foreach ($aColumns as $colname) {
-        $where .= $where ? "OR " : "WHERE ( ";
+    foreach ($aColumns as $index_col => $colname) {
+        $where .= $where ? ($cnt_columns != $index_col ? "OR " : "" ) : "WHERE ( ";
+
         if ($colname == 'name') {
-            if ($searchMethodInPatientList) { // exact search
-                $where .=
-                    "lname LIKE '$sSearch' OR " .
-                    "fname LIKE '$sSearch' OR " .
-                    "mname LIKE '$sSearch' ";
-            } else {
                 $where .= // like search
```

Comment:
is above `like search` pertinent anymore?

Reply:
Everything remained as before, just slightly improved the code.